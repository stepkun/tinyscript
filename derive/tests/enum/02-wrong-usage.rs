//! Test wrong usage of `tinyscript` enum derive macro `ScriptEnum`
// Copyright Â© 2025 Stephan Kunz

#[tinyscript_derive::ScriptEnum]
enum TestEnum {
	CaseA,
	CaseB,
}

// dummy main
fn main() {}
