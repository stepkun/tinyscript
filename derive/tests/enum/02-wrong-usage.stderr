error: expected attribute, found derive macro `tinyscript_derive::ScriptEnum`
 --> tests/enum/02-wrong-usage.rs:4:3
  |
4 | #[tinyscript_derive::ScriptEnum]
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not an attribute
